### PR TITLE
Fixes and tests

### PR DESCRIPTION
**Pull request by encukou**
A couple more fixes and tests for `naucse_render`.
I'm pushing early; I did not yet check compatibility with naucse. Planning to do that later.

### CODE REVIEW

File: test_naucse_render/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This is a package.
+
+# For a long time, pytest discouraged test packages, but no longer;  see:
+# https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code
+# This makes it easy for test_naucse.conftest to also include helpers that
```
Comment (hroncok) [inline test_naucse_render/__init__.py]:
```suggestion
# This makes it easy for test_naucse_render.conftest to also include helpers that
```

**Comment by hroncok** [comment]
Travis says "Requirement already satisfied: pytest in /home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages" but it has some old version without tmp_path. Install pyetst>=4 or something like that instead.

**Comment by hroncok** [comment]
Also some tips, however probably outside of this PR scope:

 * we should CI test 3.7 for easier update in the future
 * we should setup Windows CI

**Comment by encukou** [comment]
I'd like to release this as 1.0 (so that packages with `<1.0` in requirements.txt are frozen at the old version).

**Comment by encukou** [comment]
> Also some tips, however probably outside of this PR scope:
> 
>     * we should CI test 3.7 for easier update in the future
> 
>     * we should setup Windows CI

Filed as https://github.com/pyvec/naucse_render/issues/5

**Comment by encukou** [comment]
Thank you!

**Pull request merged**